langchain/document_loaders/sitemap.py

```diff
@@ -10,6 +10,8 @@
 def _default_parsing_function(content: Any) -> str:
     return str(content.get_text())
 
+def _default_meta_function(list: dict, _content: Any) -> dict:
```

Comment:
could we rename `list`? something like `element` or `meta` or `metadata`?

also do you have an example where _content is needed?

Reply:
i renamed the parameter to meta

for an example, i added another integration test that shows what could be done - i extracted the title text in the test